packages/patrol_cli/lib/src/crossplatform/flutter_tool.dart

```diff
@@ -171,8 +187,9 @@ class FlutterTool {
               _logger.success(
                 'Hot Restart: attached to the app\n'
                 'Patrol develop key commands:\n'
-                'r Hot restart.\n'
-                'q Quit (terminate the process and application on the device).',
+                'r Hot restart\n'
+                'h Help\n'
```

Comment:
Could be "Print this help message" instead of help. It's the usual wording of such descriptions